tests/jni_api.rs

```diff
@@ -600,13 +590,33 @@ pub fn throw_defaults() {
     test_throwable_descriptor_with_default_type(&env, JNIString::from(TEST_EXCEPTION_MESSAGE));
 }
 
+#[test]
+pub fn test_conversion() {
```

Comment:
Shan't this be tested more explicitly (it is unclear what the string length has to do with the conversion):

```
JObject src = <any object, let it be same string>
T wrapper = create_wrapper(src);
JObject actual = wrapper.into();
assert_that(is_same_instance(&env, src, actual));
```